main/src/cgeo/geocaching/connector/trackable/GeokretyAuthorizationActivity.java

```diff
@@ -0,0 +1,77 @@
+package cgeo.geocaching.connector.trackable;
+
+import cgeo.geocaching.R;
+import cgeo.geocaching.activity.TokenAuthorizationActivity;
+import cgeo.geocaching.network.Network;
+import cgeo.geocaching.settings.Settings;
+
+import ch.boye.httpclientandroidlib.HttpResponse;
+
+import android.os.Bundle;
+
+import java.util.regex.Pattern;
+
+public class GeokretyAuthorizationActivity extends TokenAuthorizationActivity {
+
+    private static final Pattern PATTERN_IS_ERROR = Pattern.compile("error ([\\d]+)");
```

Comment:
I think too they are better as local variables. I think @Bananeweizen was due to an uncertainty about whether this authentication was derived form OAuth or was completely different, and the comment may be moot now that this has cleared up.